src/App.js

```diff
@@ -1,13 +1,133 @@
 import React from 'react';
 import './App.css';
-
+import contacts from './contacts.json';
+ 
 class App extends React.Component {
+  state = {
```

Comment:
Wooow, look at you all fancy with CSS! Gotta take a few lessons with you! Very good job!